qr_web/settings.py

```diff
@@ -27,18 +24,22 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []  # type: List[str]
+ALLOWED_HOSTS = ['*']
```

Comment:
Don't rush, there will be different ticket for this

Reply:
without it pylint complains on ALLOWED_HOSTS

Reply:
add typing